### DISCUSSION TITLE
Google map is not taking language dynamically

### DISCUSSION THREAD

**Discussion by intimetec-shiv-maheshwari** [inocan-group/vue3-google-map#244] [Q&A]
I was trying to pass language to the google map component but google map is not reflecting the language change when I toggle the language.

**Comment by HusamElbashir** [comment]
This isn't supported unfortunately. We rely on `@googlemaps/js-api-loader` for the loading of the Google Maps API script which doesn't have a way currently to allow us to remove and reload the script (see https://github.com/googlemaps/js-api-loader/issues/100 for example).

You could try and implement this yourself through the `apiPromise` prop of the `GoogleMap` component. See @JoseGoncalves's example here for reference:
https://github.com/inocan-group/vue3-google-map/pull/178#issuecomment-1798363749

**Comment by JoseGoncalves** [comment]
I've published a composable to address this issue.

Please check:
* https://github.com/JoseGoncalves/vue-google-maps-loader
